app-libs/stf/src/stf_sgx.rs

```diff
@@ -184,7 +192,7 @@ impl Stf {
 				},
 				TrustedCall::balance_shield(enclave_account, who, value) => {
 					ensure_enclave_signer_account(&enclave_account)?;
-					debug!("balance_shield({:x?}, {})", who.encode(), value);
+					debug!("balance_shield({}, {})", account_id_to_string(&who), value);
```

Comment:
The `{:x?}` format did not work for the account id, it just printed the `Vec<u8>`, which was not very readable and hard to compare.